### ISSUE TITLE
Unify how pachctl represents pfs objects

### ISSUE THREAD

**Issue by jdoliner** [pachyderm/pachyderm#956]
We have a few competing ways that we represent pfs objects in pachctl.
Sometimes we use spaces to delimit them like in:

```
pachctl put-file repo commit file
```

But sometimes we use slashes like in:

```
pachctl flush-commit repo/commit repo/commit...
```

We should unify this to always using slashes, spaces do look a little bit nicer but it gets confusing if you need to reference more than one object. That's actually the reason that `flush-commit` uses slashes. Unifying to slashes should be less cognitive overhead for people and lead to fewer errors.

This is in response to an issue a user hit where he got confused and tried to do:

```
pachctl get-file repo repo/commit filename
```

Understandable how that could happen given that's how we print commits in `ListCommit`.

**Comment by JoeyZwicker** [comment]
I'm all for unifying, but just to play the other side, we could also print
the repo name and commitID in separate columns for list-commit

**Comment by jdoliner** [comment]
The issue isn't printing things in `list-commit` that one could obviously work either way. It's that commands like `flush-commit` need to reference multiple commits in different repos. If you want to play the other side then commands like that are the ones you need to offer another proposal for. The syntax of `flush-commit repo1 commit-in-repo1 repo2 commit-in-repo2` seems unpleasant to me since there's this implicit connection between different command line args. Furthermore this is a command that can only function with an even number of arguments and we'll have to give users error messages to that effect which seems kind of confusing.

**Comment by Tryneus** [comment]
This should be resolved in `v1.9.0` - objects now use a consistent format - `repo@branch:file` and `repo@commit:file`.
